tests/common.go

```diff
@@ -2763,6 +2763,61 @@ func DeleteBackupLocation(name string, backupLocationUID string, orgID string) e
 
 }
 
+// Delete Schedule Backup
+func DeleteScheduleBackup(backupScheduleName, backupScheduleUID, schedulePolicyName, schedulePolicyUID, OrgID string) (err error) {
+	var ctx context1.Context
+
+	Step(fmt.Sprintf("Delete scheduled backup %s of all namespaces on cluster %s in organization %s",
```

Comment:
Lets be specific while updating step names. Here this step is just to delete the schedule and not the backup. You may need to re-phrase to "Delete schedule %s for all namespaces on cluster %s in organization %s